cli/JSXV4.md

```diff
@@ -145,19 +147,26 @@ module FancyInput = {
     </div>
   )
 }
+```
 
-@react.component
-let make = () => {
-  let input = React.useRef(Js.Nullable.null)
-
-  <div>
-    <FancyInput ref=input> // only `ref` is allowed
-      <button onClick> {React.string("Click to focus")} </button>
-    </FancyInput>
-  </div>
+In V4, use this instead:
+
+```rescript
+module FancyInput = {
+  @react.component
+  let make = () => {
+    let input = React.useRef(Js.Nullable.null)
+
+    <div>
+      <FancyInput ref=input> // only `ref` is allowed
+        <button onClick> {React.string("Click to focus")} </button>
+      </FancyInput>
+    </div>
+  }
```

Comment:
I guess you miss-read my example. This module is not the fix of `FancyInput`

Reply:
The point of this example is that `ref` is only allowed, not `ref_` like the official doc. https://rescript-lang.org/docs/react/latest/forwarding-refs#discouraged-reactforwardref

Reply:
> The point of this example is that `ref` is only allowed, not `ref_` like the official doc. https://rescript-lang.org/docs/react/latest/forwarding-refs#discouraged-reactforwardref

OK would you fix the doc directly? Seems easier.

Reply:
https://github.com/rescript-lang/syntax/pull/652/commits/3e17c7bfef05d741c907efcabf53e0855b7526fa